src/Tools/dotnet-counters/Program.cs

```diff
@@ -119,6 +142,7 @@ private static Task<int> Main(string[] args)
         {
             var parser = new CommandLineBuilder()
                 .AddCommand(MonitorCommand())
+                .AddCommand(ExportCommand())
```

Comment:
I'd recommend adding tests and updating the [design-doc](https://github.com/dotnet/diagnostics/blob/master/documentation/design-docs/dotnet-tools.md) in the same PR where the implementation goes in. This both makes it easier to correlate later on and prevents the kind of technical debt that we built up rushing along earlier in the 3.0 release.